convert2rhel/toolopts.py

```diff
@@ -325,6 +362,19 @@ def _process_cli_options(self):
         if parsed_opts.org:
             tool_opts.org = parsed_opts.org
 
+        # Checks of multiple authentication sources
+        if tool_opts.password and tool_opts.activation_key:
```

Comment:
@r0x0d If the `tool_opts` would be changed to `parsed_opts`, there will be need for more checks, because `password` and `activation_key` can came from more sources (like config file, password file). But in the `tool_opts` is everything merged and is possible to easily check, if there are more than one verification method.